scripts/run_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [ "$1" == "state_machine" ]; then
+    ./zig/zig test src/state_machine.zig
```

Comment:
We could also run the state machine tests with `zig test src/state_machine.zig` so that we don't need to maintain a new shell script.

Reply:
I have made the script more dynamic, if still an issue I will gladly remove. :-)

Reply:
This is pretty good but let's rather drop the shell script and then later we can provide a `test` run option from our `build.zig`. That would be a better place to do this and then we won't need to mirror `.sh` scripts for Windows.

Reply:
Removed.